src/execution.jl

```diff
@@ -56,6 +73,14 @@ This is a low-level call, prefer to use [`cudacall`](@ref) instead.
     _launch(f, griddim, blockdim, shmem, stream, args)
 end
 
+@inline function launch{N}(f::CuFunction, griddim::CuDim3, blockdim::CuDim3,
```

Comment:
Also, maybe make that `launch` accept `kwargs`, then it would mimic `cudacall` which has both a `kwargs` version, and a non-`kwargs` one for performance.